app/Models/Location/Country.php

```diff
@@ -49,7 +49,7 @@ public function getTotalLitterRedisAttribute ()
     {
         return Redis::hexists("country:$this->id", "total_litter")
             ? (int)Redis::hget("country:$this->id", "total_litter")
-            : 0;
+            : 1;
```

Comment:
Revert this too, add checks where it's used. I think this can be simplified and we can remove that `hexists`, but leave it for now and just revert 1 to 0.